accepted/future-releases/constructor-tearoffs/feature-specification.md

```diff
@@ -735,3 +736,4 @@ In this case, most of the parameters are *unnecessary*, and a tear-off expressio
 * 2.14: Remove many disambiguation tokens. Allow instantiating function *objects* and *callable objects*. Mention forwarding constructors from mixin applications.
 * 2.15: Add section about constants and specify new rules about potentially constant and constant expressions of the form <code>e\<T<sub>1</sub>..T<sub>k</sub>></code>.
 * 2.16: Add one more kind of potential constant, type parameters.
+* 2.17: Specify that <code>*e*\<*typeArgs*></code> can desugar to <code>*e*.call\<*typeArgs*></code> when `call` is an extension method.
```

Comment:
Also update the current version at the top. It's been stuck on 2.14.

Reply:
Ah, yes! Done.